api/src/utils/task-audit-record.test.js

```diff
@@ -4,7 +4,9 @@ describe('TaskAuditRecord', () => {
   describe('updateTaskRecord', () => {
     it('creates a new task record', async () => {
       // create mock create function
-      const mockCreateFunction = jest.fn(() => Promise.resolve({ _id: '123', created: true }));
+      const mockCreateFunction = jest.fn(() =>
+        Promise.resolve({ _id: '123', created: true, read: ['sysadmin'], write: ['sysadmin'] })
```

Comment:
For the purpose of the test, it doesn't actually matter what we return, it could be a sentence even.  The real call to mongoose/mongo is being mocked out, so the ```_id: '123'``` is just there to provide evidence that the code called the mongoose function like we expected it to.  I only use _id because its recognizable and gives you some extra info about what kinds of things the test is doing.